packages/create-plugin/templates/common/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
     "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
     "test": "jest --watch --onlyChanged",
-    "test:ci": "jest --maxWorkers 4",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
```

Comment:
> While It's really lovely to force testing, if you're new to writing a plugin and just following the commands in the readme to verify you have everything up and running it can be a bit overwhelming to read and easy to think you've done something "wrong" and to know what to do to fix it:

I'm not opposed to adding the `--passWithNoTests` flag here but I think this has highlighted  an oversight with the templates used to scaffold a datasource. I believe we should include a copy of [this placeholder test file](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/panel/src/module.test.ts). At least then we're consistently providing a test file for all three plugin types (even if the dev decides to ignore or delete them). WDYT @sarahzinger ?

Reply:
Sure I think either way works!